src/components/Clock.tsx

```diff
@@ -0,0 +1,71 @@
+import { Component } from 'react';
+import './Clock.scss';
+import cn from 'classnames';
+
+interface Props {
+  name: string
+}
+
+interface State {
+  time: Date
+}
+
+export class Clock extends Component<Props, State> {
+  state = {
+    time: new Date(),
```

Comment:
Keep and set in state already transformated time
thats why your tests fail
```suggestion
    time: new Date().toUTCString().slice(-12, -4),
```